decidim-elections/spec/types/decidim/elections/elections_type_spec.rb

```diff
@@ -44,6 +46,14 @@ module Elections
             expect(response["election"]).to be_nil
           end
         end
+
+        context "when the election belongs to the component and its publication time is nil" do
+          let!(:election) { create(:election, component: model, published_at: nil) }
```

Comment:
Here, you could also use the trait `unpublished` instead of writing `published_at: nil`.

Reply:
I tried that, but for some reason it doesn't work

Reply:
Fixed putting `unpublished` as default